openedx_learning/apps/authoring/backup_restore/zipper.py

```diff
@@ -258,18 +274,18 @@ def get_latest_modified(self, versions_to_check: List[PublishableEntityVersion])
                 latest = version.created
         return latest
 
-    def create_zip(self, path: str) -> None:
+    def create_zip(self) -> None:
```

Comment:
Please leave this as it was. The path var isn't necessary for initialization, and I think it makes sense to keep it scoped to the point where we actually want to output the file.